src/main/java/com/google/sps/ColorUtils.java

```diff
@@ -0,0 +1,143 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+import java.awt.Color;
+import java.util.ArrayList;
+
+/**
+ * Utility class for converting RGB color values to color name.
+ */
+public class ColorUtils {
+  public static final String NO_MATCHED_COLOR_MESSAGE = "No matched color name.";
+
+  /**
+   * Compiles a list of known colors.
+   */
+  private ArrayList<ColorName> initColorList() {
```

Comment:
We don't need to initialize this every time. You could define something like:

private static final ImmutableList<ColorName> COLOR_LIST;
static {
  // your code
}